app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -119,9 +119,9 @@ private Bitmap replaceColors(Bitmap myBitmap, int keepCr, int keepCg, int keepCb
                 allpixels[i] = replaceColor;
             }
         }
-
-        myBitmap.setPixels(allpixels, 0, myBitmap.getWidth(), 0, 0, myBitmap.getWidth(), myBitmap.getHeight());
-        return myBitmap;
+        Bitmap clone = Bitmap.createScaledBitmap(myBitmap, myBitmap.getWidth(), myBitmap.getHeight(), false);
```

Comment:
I think the goal is to create a separate Bitmap, but using `createScaleBitmap` without needing to rescale the bitmap can't be the best answer; that also makes `setPixels` redundant. I think we want to create an empty bitmap and then use `setPixels`, as in (untested):
```java
Bitmap clone = Bitmap.createBitmap(myBitmap.getWidth(), myBitmap.getHeight(), myBitmap.getConfig());
clone.setPixels(allpixels, 0, myBitmap.getWidth(), 0, 0, myBitmap.getWidth(), myBitmap.getHeight());
```